src/web/components/SignInGate/gateDesigns/SignInGateMain.tsx

```diff
@@ -86,12 +89,13 @@ export const SignInGateMain = ({
 				</LinkButton>
 			</div>
 
-			<p className={cx([bodyBold, signInHeader])}>
+			<p className={cx([bodySeparator, bodyBold, signInHeader])}>
```

Comment:
There's the `DividerBlockComponent` that could perhaps be used here?

```
type Props = {
	size?: 'full' | 'partial';
	spaceAbove?: 'tight' | 'loose';
};
```

This component is a candidate for being promoted up to Source with the idea to share code and designs so that we have one homogenous way to divide content.

Reply:
I'll give it a go to see if it works with the design 👍